### ISSUE TITLE
web3.utils.isAddress appears to return a string not a boolean

### ISSUE THREAD

**Issue by epasquali** [ethereum/web3.js#2163]
There is an error in the documentation, 

https://web3js.readthedocs.io/en/1.0/web3-utils.html#isaddress

The documentation claims a call to web3.utils.isAddress() returns a boolean value, however this is NOT true, the return value is a string. Hence anyone coding checking for the boolean (for instance, by doing something like:

if (web3.utils.isAddress('asdased'))
{
   //something here
}

will have unexpected results. I propose someone change the documentation to reflect he real return type, which is a string as evidenced by simply running a typeof(web3.utils.isAddress('abcd')) call.
(it is indeed a string).

An even better solution would be to return a real boolean, and not a string.

**Comment by Flash-Git** [comment]
What does the returned string contain? I'm unfortunately not able to test this right now but that sounds very odd.

EDIT: It returns a boolean for me. Can you post more information?

**Comment by dev1644** [comment]
Yeah, It returns a boolean value, not a String.

**Comment by epasquali** [comment]
Strange, it returns a string for me. Checked it several times. I'm using the web3.js from the CDN.

**Comment by dev1644** [comment]
I have tested this function in both CDN and node environment, its returning boolean value.
Post the CDN that you are using, let's check it.

**Comment by epasquali** [comment]
Ok, I figured out the error. 

I was doing:

status = web3.utils.isAddress('abcd')

then typeof(status) returned a string.

When I did

var status = web3.utils.isAddress('abcd')

typeof(status) was indeed a boolean.

Sorry I'm a javascript noob and this makes no sense to me.

But have confirmed typeof(web3.utils.isAddress('abcd')) is indeed boolean using https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.0.0-beta.36/dist/web3.min.js
